src/interface_r/README.md

```diff
@@ -1,32 +1,142 @@
+
+
+## Troubleshooting
+
+If you have any issues, or have any recommendations to the installation instructions, please let us know by filing a GitHub issue.  If there are installation issues, the first thing to check is the [system requirements](https://github.com/h2oai/h2o4gpu#requirements).
+
+
+### GPUs
+
+The first thing to check is that you actually have a machine with Nvidia GPUs.  If you can run the `nvidia-smi` command in the shell and get an output that looks similar to this, it means you indeed have GPUs and working drivers on your machine:
+
+```
+username@gpubox:~$ nvidia-smi
+Tue Mar 27 11:38:14 2018
++-----------------------------------------------------------------------------+
+| NVIDIA-SMI 387.34                 Driver Version: 387.34                    |
+|-------------------------------+----------------------+----------------------+
+| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
+| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
+|===============================+======================+======================|
+|   0  GeForce GTX 1080    On   | 00000000:02:00.0 Off |                  N/A |
+| 27%   39C    P8    10W / 180W |     12MiB /  8112MiB |      0%      Default |
++-------------------------------+----------------------+----------------------+
+|   1  GeForce GTX 1080    On   | 00000000:81:00.0 Off |                  N/A |
+| 27%   39C    P8    11W / 180W |     12MiB /  8114MiB |      0%      Default |
++-------------------------------+----------------------+----------------------+
+
++-----------------------------------------------------------------------------+
+| Processes:                                                       GPU Memory |
+|  GPU       PID   Type   Process name                             Usage      |
+|=============================================================================|
+|  No running processes found                                                 |
++-----------------------------------------------------------------------------+
+```
+
+
+
+### CUDA 
+
+To check if CUDA 8 or 9 is installed, run the `nvcc --version` command.  If you see this:
+
+```
+-bash: nvcc: command not found
+```
+That means that CUDA is not installed.  If you do have CUDA installed, you will see something like this:
+
+```
+nvcc: NVIDIA (R) Cuda compiler driver
+Copyright (c) 2005-2016 NVIDIA Corporation
+Built on Tue_Jan_10_13:22:03_CST_2017
+Cuda compilation tools, release 8.0, V8.0.61
+```
+
+
+### Python issues
+
+If you try to train a model and you get a non-descript error like this:
+
+```
+> # Initialize and train the classifier
+> model <- h2o4gpu.random_forest_classifier() %>% fit(x, y)
+Error: 
+```
+
+Or if you receive a full error like this:
+
+```
+Error: Python module h2o4gpu was not found.
```

Comment:
same here missing `eval = FALSE`? I wonder if there's global option for this instead of specifying for every block of code.

Reply:
This is just the README.md, not the vignette, so we don't need `eval = FALSE`.